lib/kits/paged-publication/bootstrapper.ts

```diff
@@ -18,7 +18,11 @@ export default class Bootstrapper {
     }
 
     createViewer(
-        data: {details: V2Catalog; pages: V2Page[]},
+        data: {
+            details: V2Catalog;
+            pages: V2Page[];
+            pageDecorations: V2PageDecoration[];
```

Comment:
This should be optional, as someone might intentionally want to start a viewer without decorations(for example to implementation their own link decorations outside of PagedPublicationKit, like eta.dk might do)